src/kirby/components/list/list.component.html

```diff
@@ -1,7 +1,28 @@
 <ng-container *ngTemplateOutlet="headerTemplate"></ng-container>
+
 <ul>
-  <li *ngFor="let item of items" (click)="onItemClick(item)">
-    <ng-container *ngTemplateOutlet="listItemTemplate; context: {$implicit: item}" >
-    </ng-container>
-  </li>
-</ul>
\ No newline at end of file
+
+  <ng-container *ngIf="isSectionsEnabled; else normalList">
+      <ng-container *ngFor="let section of items | sectionBy:getSection">
```

Comment:
A bit easier to read like this:
```suggestion
      <ng-container *ngFor="let section of (items | sectionBy:getSection)">
```